### PR TITLE
Add CLEVA Schema

### PR DESCRIPTION
**Pull request by lyy1994**
This PR adds CLEVA schema to `schema.yaml` according to the scaffolding of this HELM [PR](https://github.com/stanford-crfm/helm/pull/1845).

### CODE REVIEW

File: src/helm/benchmark/static/schema.yaml [hunk 1]
```diff
@@ -2944,540 +2974,560 @@ run_groups:
...
+      who: n/a
+      when: n/a
       language: Chinese
 
-  - name: cleva_classical_chinese_understanding
-    display_name: CLEVA (Chinese) classical Chinese understanding
-    description: CLEVA (Chinese) classical Chinese understanding
+  - name: cleva_summarization
+    display_name: CLEVA (Chinese) summarization
+    description: Summarize a dialogue between a customer representative and a customer.
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: We should specify that dialogue summarization is just a subtask and there are a lot of summarization tasks we may include in the future that could summarize any form of text.

File: src/helm/benchmark/static/schema.yaml [hunk 2]
```diff
@@ -3005,14 +3004,14 @@ run_groups:
       main_split: test
     taxonomy:
       task: summarization
-      what: "?"
-      who: "?"
-      when: "?"
+      what: e-commercial
+      who: customers and representatives
+      when: n/a
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml:3009]: Maybe an unknown "?" is more appropriate?

File: src/helm/benchmark/static/schema.yaml [hunk 3]
```diff
@@ -3005,14 +3004,14 @@ run_groups:
       main_split: test
     taxonomy:
       task: summarization
-      what: "?"
-      who: "?"
-      when: "?"
+      what: e-commercial
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: "e-commercial" does not seem to be some form of text.

File: src/helm/benchmark/static/schema.yaml (hunk 2 above)
Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml:3009]: I think "?" stands for unknown and "n/a" means this field is not necessary. In most cases, I believe we just don't know "who" wrote "what" at "when", but it does not mean that these fields are not needed. Correct me if I am wrong.

File: src/helm/benchmark/static/schema.yaml [hunk 4]
```diff
@@ -3057,14 +3056,14 @@ run_groups:
...
-      when: "?"
+      what: product description
+      who: n/a
+      when: n/a
       language: Chinese
 
   - name: cleva_dialogue_generation
     display_name: CLEVA (Chinese) dialogue generation
-    description: CLEVA (Chinese) dialogue generation
+    description: Task-oriented dialogue between a user and a system.
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: Similar to summarization, we may just say this is a general task and currently it contains a subtask focused on the more specific task-oriented dialogues.

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml]: So does the following opinion mining task.

File: src/helm/benchmark/static/schema.yaml [hunk 5]
```diff
@@ -3290,9 +3288,9 @@ run_groups:
       main_split: test
     taxonomy:
       task: multiple-choice
-      what: "?"
-      who: "?"
-      when: "?"
+      what: Chinese idioms
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: "Chinese idioms" may be somewhat misleading as they are passages from novels, essays, and news articles.

File: src/helm/benchmark/static/schema.yaml [hunk 6]
```diff
@@ -3235,14 +3233,14 @@ run_groups:
       main_split: test
     taxonomy:
       task: multiple-choice
-      what: "?"
-      who: "?"
-      when: "?"
+      what: books
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: "contemporary literary works in Chinese" from the original paper.

File: src/helm/benchmark/static/schema.yaml [hunk 7]
```diff
@@ -3182,8 +3180,8 @@ run_groups:
       main_split: test
     taxonomy:
       task: language modeling
-      what: web text
-      who: "?"
+      what: Wikipedia and websites
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: "websites" could be somewhat misleading?

File: src/helm/benchmark/static/schema.yaml [hunk 8]
```diff
@@ -3126,15 +3124,15 @@ run_groups:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: paraphrase identification
+      what: finance or chitchat
```
Comment (lyy1994) [inline src/helm/benchmark/static/schema.yaml]: "finance" is confusing.

File: src/helm/benchmark/static/schema.yaml (hunk 7 above)
Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml]: Do you think web text is a better phrase?

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml]: Maybe. I think we should specify what kind of **text** in this scenario, not their sources.

Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml]: So how about Wikipedia and news?

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml]: Looks good to me.

File: src/helm/benchmark/static/schema.yaml (hunk 2 above)
Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml:3009]: I understand your point of view, and I will gladly accept it if you insist. In my opinion, I feel like "?" means this field is missing and will be filled later after some examinations, while "n/a" means the creation time of the data does not matter too much, and the data is not time-sensitive.

I can do either way. Please let me know what you think. Thanks.

Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml:3009]: I can also change the value to something like "not specified" to deliver a more clear indication.

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml:3009]: Perhaps we can stay this way and let the HELM team decide.

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml:3009]: They should have some guidelines on how to fill out this `schema.yaml` in the future.

Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml:3009]: Sure 😊

**Comment by Jianqiao-Zhao** [comment]
I push my code to reflect all the discussions above. Have a look. Thanks!

**Review by lyy1994 — COMMENTED** [review]
I have no other comments. Good job! I think this PR is ready to be merged and @Jianqiao-Zhao could review other parts. Thanks.

**Review by Jianqiao-Zhao — APPROVED** [review]

**Pull request merged**